gokay/length_test.go

```diff
@@ -29,3 +29,38 @@ func TestLengthSlice(t *testing.T) {
 	var actual = int64(99)
 	require.NoError(t, LengthSlice(expected, actual))
 }
+
+func TestMinLengthString(t *testing.T) {
+	var validCases = []struct {
+		String    string
+		MinLength int64
+	}{
+		{String: "foo", MinLength: 3},
+		{String: "", MinLength: 0},
+		{String: "a", MinLength: 1},
+		{String: "a", MinLength: 100},
```

Comment:
Length was 1, needs to be at least 100, len(*str), expected
I think it's picking up the address of the pointer to check